dags/atd_finance_data_fdus.py

```diff
@@ -0,0 +1,154 @@
+# test locally with: docker compose run --rm airflow-cli dags test atd_finance_data_fdus
```

Comment:
side note that I couldn't test locally because `atddocker/atd-finance-data:production` is not multi-architecture. Not saying that this should be because it will run on atd-data03 as-is.